recipes/foamalgo/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "foamalgo" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "135af3991e10d2275f1ad6ff68d1a497e3ad9b7b6cba718d551d1d07cb0e23d7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zhujun98/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
```

Comment:
Does this really use cmake?

Reply:
@scopatz Thanks for the review! Yes, it uses cmake in `setup.py`. Actually, I would like to install the C++ library (head-only) and Python binding at the same time. But I have no experience of this. So for now, I have not put the cmake install in `build.sh`.

Reply:
I believe you should use outputs then, because we can separate those packages
Please see the documentation for more information about that
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#outputs-section

Reply:
I finally gave up outputs because of the wired errors. The same setup works at least on linux. Thanks anyway!